web/app/scripts/recent-counts/recent-counts-controller.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function RecentCountsController($scope, RecordAggregates) {
+        var ctl = this;
+        init();
+        $scope.$on('driver.state.recordstate:selected', init);
+        return ctl;
+
+        function init() {
+            /* jshint camelcase: false */
+            RecordAggregates.recentCounts().then(function(aggregate) {
+                ctl.record_type_plural = aggregate.plural;
```

Comment:
Seems a little weird that plural is being added to/obtained from the aggregate. Any reason you didn't just grab it from the record type on the client-side?